### DISCUSSION TITLE
Create repo on mobile with app

### DISCUSSION THREAD

**Discussion by max-dmg** [community/community#4316] [Mobile]
Why is creating a repo with the Mobile App not a feature?

**Comment by benhatsor** [comment]
A related project is [codeit.codes](https://codeit.codes), from which you create and edit GitHub repos on mobile.

**Comment by HenSquared** [comment]
You can now create repositories on iOS and Android :) 
https://github.blog/changelog/2026-05-11-create-repositories-on-the-go-with-github-mobile/
